tests/auth_tests/test_forms.py

```diff
@@ -209,6 +212,28 @@ class Meta(UserCreationForm.Meta):
         form = CustomUserCreationForm(data)
         self.assertTrue(form.is_valid())
 
+    def test_custom_form_hidden_username_field(self):
+        class CustomUserCreationForm(UserCreationForm):
+            class Meta(UserCreationForm.Meta):
+                model = CustomUserWithUUIDAsUsername
+                # We deliberately left USERNAME_FIELD outside
+                fields = ('email',)
+
+            def save(self, commit=True):
```

Comment:
Yes, it's definitely not needed. Updated the PR to re-use `CustomUserWithoutIsActiveField` instead.